spec/models/user_spec.rb

```diff
@@ -1,5 +1,49 @@
 RSpec.describe User do
   describe 'バリデーションのテスト' do
+    context 'nameが10文字以内の場合' do
+      let(:user) { build(:user, name: 'hoge_hoge_') }
+
+      it 'valid?がtrueになる' do
+        expect(user.valid?).to eq true
```

Comment:
IMO

errorsが空文字であることもテストしよう。
```suggestion
      aggregate_failures do
        expect(user.valid?).to eq true
        expect(user.errors.full_messages).to eq []
      end
```

Reply:
修正しました
[267f050](https://github.com/ok-os-job-change-team/kaito-twitter-clone-bootcamp/pull/87/commits/267f0505835a1d9bee1cc92c498d81594c922638)